packages/babel-plugin-transform-react-jsx/test/fixtures/react/.optimisation.react.constant-elements/expected.js

```diff
@@ -1,40 +0,0 @@
-var _ref = React.createElement(
-  "div",
```

Comment:
The output in the reenabled test is identical, only it's still in JSX instead of transformed to React.createElement calls.